daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -364,78 +361,6 @@ packagingTests = testGroup "packaging"
               responseBody queryResponse @?= "{\"result\":{\"identifier\":\"Alice\",\"isLocal\":true},\"status\":200}"
               -- waitForProcess' will block on Windows so we explicitly kill the process.
               terminateProcess ph
-    , testCase "daml ledger via json api" $ withTempDir $ \tmpDir -> do
```

Comment:
Do we still have a test here that makes sure that `daml ledger` properly forwards commands to `daml-helper ledger`? Otherwise, I'd be a bit scared that we mess something up in the daml assistant logic for forwarding commands and won't notice. Testing for a single `daml ledger` subcommand should be enough and then we can test all the various `daml ledger` subcommands in the other test file.

Reply:
There are currently no `daml ledger` tests in the assistant integration tests. I now left one test in there that simply calls the `daml ledger allocate-party` command to test the plumping. It's quite the effort though (each such test takes around 15 secs for starting sandbox and JSON HTTP server).

Reply:
> It's quite the effort though (each such test takes around 15 secs for starting sandbox and JSON HTTP server).

Yeah, that's why I asked only for one single test. Thanks for adding that one.